src/validate.ts

```diff
@@ -189,6 +194,9 @@ export function response(compiledPath: CompiledPath | undefined,
   if (response === undefined) {
     response = operation.responses['default'];
   }
-
+  // make Date or ObjectId to string
```

Comment:
I think there are two issues: 1) there's a performance overhead introduced where the parse/stringify step is not needed and 2) we're baking in an assumption about response serialization - which is fine in the case of koa/express/etc but may not be valid for all use cases. 

My preference would be to add it as part of an options object (serialize, defaults to false) passed to swagger2-koa `validate`.  Happy to do it if you want to raise an issue on that project.